README.md

```diff
@@ -51,6 +52,15 @@ If you are already using a specific FL framework, you can port your code to work
 
 Please find a full documentation of this project [**here**](docs/README.md).
 
+### Real-world example benchmarks
+
+A benchmarking analysis is performed for each real-world example to understand the validy, efficiency and scalability of our implementation of FL:
+
+|  Training overhead  | Model performance  | Scalability |
+|:-------------------:|:------------------:|:-----------:|
+| [![overhead icon](./docs/pics/pneumonia_time.jpg)](./docs/concepts/benchmarking.md/#21-training-overhead)| [![performance icon](./docs/pics/pneumonia_acc.jpg)](./docs/concepts/benchmarking.md/#22-model-performance)| [![scala icon](./docs/pics/pneumonia_ddp.jpg)](./docs/concepts/benchmarking.md/#23-scalability-with-training)
```

Comment:
nit: maybe we can keep the dimensions of these images the same as real-world examples images

Reply:
Thanks for the suggestion, Amit! It looks like the figure sizes adapted automatically to the table size. There might be a way to make them smaller but maybe that would be too small for these figures given we have some labels and legends here.  What do you think? Always happy to make the change though😉

Reply:
oh, I see! let's keep it as it is then